### ISSUE TITLE
5.4 is released!

### ISSUE THREAD

**Issue by jpmurray** [corcel/corcel#240]
All right, I'm quick on the trigger, but Laravel 5.4 has just been tagged, and Corcel isn't compatible with it because of requirements

```
"illuminate/database": "~5.3.0",
"illuminate/contracts": "~5.3.0",
"illuminate/support": "~5.3.0",
```

Do you think it would be as easy as changing those for 5.4 to make it play with the new release?

**Comment by jgrossi** [comment]
@jpmurray wow you're fast! LOL I'm working in making Corcel compatible with `5.0+`, so I don't think is good for now restrict Corcel to 5.4. let's wait a few more days until the `5.0+` stuff is ready, and we can take care of the `5.4` version. in the meanwhile can you install Corcel in a `5.4` env to make sure everything is working? just change the `composer.json` content locally and test it, please :-) thanks! JG

**Comment by jpmurray** [comment]
HA! Yeah, I was waiting for the release to be tagged to start working on a refactoring-from-the-ground-up project using Corcel :)

All right, I'l be testing in the coming hour or so, will be reporting here :)

**Comment by jpmurray** [comment]
nope :(

```
Using version ^1.3 for jgrossi/corcel
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install jgrossi/corcel v1.3.3
    - Conclusion: don't install jgrossi/corcel v1.3.2
    - Conclusion: don't install jgrossi/corcel v1.3.1
    - Conclusion: remove laravel/framework v5.4.0
    - Installation request for jgrossi/corcel ^1.3 -> satisfiable by jgrossi/corcel[v1.3.0, v1.3.1, v1.3.2, v1.3.3].
    - Conclusion: don't install laravel/framework v5.4.0
    - jgrossi/corcel v1.3.0 requires illuminate/database ~5.3.0 -> satisfiable by illuminate/database[v5.3.0, v5.3.16, v5.3.23, v5.3.4].
    - don't install illuminate/database v5.3.0|don't install laravel/framework v5.4.0
    - don't install illuminate/database v5.3.16|don't install laravel/framework v5.4.0
    - don't install illuminate/database v5.3.23|don't install laravel/framework v5.4.0
    - don't install illuminate/database v5.3.4|don't install laravel/framework v5.4.0
    - Installation request for laravel/framework 5.4.* -> satisfiable by laravel/framework[v5.4.0].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by jgrossi** [comment]
@jpmurray did you install it using VSC? I mean, download the last copy of Corcel, change the `composer.json` to Laravel packages `~5.4.0`, add the VSC (in `composer.json`) to point to your local copy of Corcel and `composer update`?

**Comment by jpmurray** [comment]
Well, never mind me! I actually tested this morning while it was not fully tagged, and it wasn't working. Probably some of the dependencies where not updated yet because now it works just fine!

I'll close this and send a PR :)

**Comment by jgrossi** [comment]
@jpmurray great! but at least for now let's not force `5.4` until the `5.0+` support is finished. if using 5.4 worked well maybe just making composer using `>=5.0.0` should work, but I'd like to test one by one to make sure.

**Comment by jpmurray** [comment]
Yep, you're right, I changed it for `>=5.3.0`since your master branch has dependencies fixed at 5.3.0. Don't feel forced to merge anyway, just letting the PR there might help people who want to get it anyway :)
